test/integration/user_can_check_out_to_place_an_order_test.rb

```diff
@@ -28,8 +28,8 @@ class AnExistingUserCanCheckOutToPlaceAnOrderTest < ActionDispatch::IntegrationT
     assert_equal '/orders', current_path
 
     within('table') do
-      assert page.has_content?('ordered')
-      assert page.has_content?('View Order')
+      assert page.has_content?('Pending')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.